build-tools/packages/build-tools/src/fluidBuild/tasks/leaf/miscTasks.ts

```diff
@@ -198,10 +197,7 @@ export class GenVerTask extends LeafTask {
 
 export class TypeValidationTask extends LeafWithDoneFileTask {
 	protected async getDoneFileContent(): Promise<string | undefined> {
-		const details = await getPackageDetails(this.package.directory);
-		const content =
-			JSON.stringify(this.package.packageJson) + JSON.stringify(details.typeValidation);
-		return content;
+		return JSON.stringify(this.package.packageJson);
```

Comment:
I think I missed this when I externalized the config. Should the external config file be included here?

Reply:
I removed support for the external config, but I think you got this right. If there was external config, I think the relevant part ended up in details.typeValidation.